### PR TITLE
Added HTML/CSS Assignment

### PR DESCRIPTION
**Pull request by pradnyalgandhi**
Review HTML/CSS Assignment

### CODE REVIEW

File: HTML_CSS/README.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+## HTML/CSS challenge assignments for Wealth42
+
+### Introduction
+The goal of this challenge is to test your familiarity with concepts in HTML & CSS.
+To read more about the recruitment process and open opportunities at Wealth42, click [here](http://bit.ly/w42-hiring)
+
+### Pick an evaluation mode
+
+1.   **Project of your choice**: You are free to submit a project of your own with *some* conditions.
+     *   Qualifying criteria for submission:
```
Comment (dwividha) [inline HTML_CSS/README.md:17]: Lets do the following changes:
- remove qualifying and additional criteria
- move all criteria points as direct bullet points of 'project of your choice'

File: HTML_CSS/README.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+     *   You can select any one of the stories mentioned in the [stories](../stories/) folder, and develop a HTML Website using CSS for it. 
+     *   Follow [Instructions](#instructions) and [Submission Process](#submission-process) to complete the assignment.
+     *   For success, ensure you get as many checks as in our [Assessment Criteria](#assessment-criteria)
+
+### Instructions
+
+*   HTML & CSS: 
+    *   Create upto the mark website based on the user story selected.
+    *   Design and Organise your webpage to depict the frontend and objectives of the story selected.
+    *   Integrate CSS to add styling to the webpage.(Frameworks can be used)
```
Comment (dwividha) [inline HTML_CSS/README.md]: (Using frameworks is encouraged, however, you must also display excellence in pure CSS as well)

File: HTML_CSS/README.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+*   View: 
+    *   Design reusable components that are used across all the screens relevant
```
Comment (dwividha) [inline HTML_CSS/README.md]: HTML does not have reusable components.
Let's also add these to the requiements:
- using CSS based animations
- showcasing media queries and breakpoints to design responsive behaviour

File: HTML_CSS/README.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+2.   **Wealth42's drafted assignment**: 
+     *   You can select any one of the stories mentioned in the [stories](../stories/) folder, and develop a HTML Website using CSS for it. 
+     *   Follow [Instructions](#instructions) and [Submission Process](#submission-process) to complete the assignment.
+     *   For success, ensure you get as many checks as in our [Assessment Criteria](#assessment-criteria)
+
+### Instructions
+
+*   HTML & CSS: 
+    *   Create upto the mark website based on the user story selected.
```
Comment (dwividha) [inline HTML_CSS/README.md]: Let's replace this with - design 4 main pages, along with a placeholder page to which you can link the rest of the website (with an under construction sign)

File: HTML_CSS/README.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+    *   Design and Organise your webpage to depict the frontend and objectives of the story selected.
+    *   Integrate CSS to add styling to the webpage.(Frameworks can be used)
+*   View: 
+    *   Design reusable components that are used across all the screens relevant
+    *   Include any collateral/stock images (make sure these are okay for public use)
+    *   Include icons/vector art wherever relevant.
+*   Functionality(optional): 
+    *   Use JavaScript to add functionality to the website.
```
Comment (dwividha) [inline HTML_CSS/README.md]: * Use Javascript to make the page interactive - however, no backend integrations, only static designs are required

File: HTML_CSS/README.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+
+### Assessment Criteria
+1. Maintainability and organisation of code 
+2. Usage of
+    1. HTML, CSS or any Framework.(Mention the Framework in the README.md)
+3. Balancing between speed and quality of code - we understand that candidates can only spend limited time on these challenges.
+If you do make any trade-offs in favour of saving time, do mention them in comments.  
+4. Ownership - going beyond the exact job description to deliver a finished product.
+For a HTML/CSS dev, this could translate to: 
+    1. Creating additional functionality using JavaScript.
```
Comment (dwividha) [inline HTML_CSS/README.md]: 1- Making smooth animations between different transitions
2- using flexbox and grid techniques separately to show mastery in both

**Review by dwividha — CHANGES_REQUESTED** [review]

**Comment by pradnyalgandhi** [comment]
Requested changes are done.
You can review it and merge the pull request.
Thanks

**Pull request closed without merging**